.github/actions/install-tkn/action.yaml

```diff
@@ -0,0 +1,10 @@
+name: Install tkn
```

Comment:
Why not to use an existing one? https://github.com/marketplace/actions/install-tkn

Reply:
The main reason is that the goal here is to do the same as we do in https://github.com/redhat-appstudio/release-service-bundles . 

And why don't we use the existing action there? We used to, but some time ago we moved to installing it directly, since it's only two lines of code and the community action hadn't been updated in a long time: https://github.com/redhat-appstudio/release-service-bundles/pull/109

Since then we moved it to a local action definition since it is used in two places in that repo. I could run it directly in this repo (without the action definition), but I think it's better to stay consistent and have it this way.

Reply:
Does the action needs to be updated? I checked the code and the url still seems to be valid and you can customize the version. So why to maintain a new action when there's one providing the same functionality?

The other use case is slightly different because you don't define a new action, but I would still use the same existing action for that.

Reply:
> So why to maintain a new action when there's one providing the same functionality?

Because it's fairly trivial. Depending on somebody else's action always brings risks with it. When the action is provided by github, that gives it some trustworthiness. It's acceptable to use community provided actions if they provide value, but in this case I just don't see the benefit.

But that's my opinion. If you think I should use the action, I can change it. Let me know.

Reply:
My opinion is that if there's an action that provides the feature, we should not reinvent the wheel and just use it. Now, if there's a valid reason for not to do it, that's a different story. I just don't see any benefit on writing our own.

If you have a strong opinion on this, go for it. If not, I'd say is better to use existing stuff. Your call, sir ;)

Reply:
@davidmogar I changed it to use the community action. Please check now.